d2l-input-time.html

```diff
@@ -19,10 +19,6 @@
 	<template strip-whitespace>
 		<style include="d2l-input-styles">
 			:host {
-				--d2l-input-line-height: 1;
```

Comment:
Removing so the date and time input are the same heights.  Date was updated to match the heights of other inputs, but I missed adjusting the time styling.